### ISSUE TITLE
[BITV]: Wrong attributes `aria-label` on `<span>`

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/photos#2288]
To Reproduce

1. Go to photos/albums
2. Create new album
3. Click to "Add"
4. File Picker opens: discover checkboxes (checkbox-radio-switch) and see that `<span>` has an  `aria-label`.

To fix: `aria-label` should be on `<input>` element like everywhere else for checkbox-radio-switch elements

![Screenshot from 2024-01-26 10-45-15](https://github.com/nextcloud/photos/assets/6078378/d275b8a0-134d-48f8-87d1-a2ec55996e55)
